From cf506772e8318529e04477f51677202f16b385d6 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Fri, 5 Dec 2014 11:37:36 -0500
Subject: [redhat] kabi: Adding symbol truncate_inode_pages (mm/truncate.c)

Message-id: <1417779456-22800-1-git-send-email-poros@redhat.com>
Patchwork-id: 101155
O-Subject: [PATCH RHEL7.1] [redhat] kabi: Adding symbol truncate_inode_pages (mm/truncate.c)
Bugzilla: 1071099

Symbol already acked. This is repost, because checksum has been changed.
    Also added for ppc64le.

    Bugzilla: 1071099
    https://bugzilla.redhat.com/show_bug.cgi?id=1071099

    Description:
    ============
    Truncate *all* the pages from an offset.

Signed-off-by: Petr Oros <poros@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/truncate_inode_pages b/redhat/kabi/kabi-module/kabi_ppc64/truncate_inode_pages
new file mode 100644
index 0000000..22561dc
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/truncate_inode_pages
@@ -0,0 +1,2 @@
+#1-
+0x59ca9631 truncate_inode_pages vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_ppc64le/truncate_inode_pages b/redhat/kabi/kabi-module/kabi_ppc64le/truncate_inode_pages
new file mode 100644
index 0000000..f23d725
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64le/truncate_inode_pages
@@ -0,0 +1,2 @@
+#1-
+0xea474e78 truncate_inode_pages vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/truncate_inode_pages b/redhat/kabi/kabi-module/kabi_s390x/truncate_inode_pages
new file mode 100644
index 0000000..c1b7bfe
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/truncate_inode_pages
@@ -0,0 +1,2 @@
+#1-
+0x41dd6192 truncate_inode_pages vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/truncate_inode_pages b/redhat/kabi/kabi-module/kabi_x86_64/truncate_inode_pages
new file mode 100644
index 0000000..52926e6
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/truncate_inode_pages
@@ -0,0 +1,2 @@
+#1-
+0x554a5293 truncate_inode_pages vmlinux EXPORT_SYMBOL
-- 
1.7.1